cityofaustin/atd-moped
# 18390 add toolbar to lexical

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/18390

## Testing
**URL to test:** 

local

**Steps to test:**
- add a project note, try enabling and disabling all of the different styling options on different text selections and confirm the styling is being applied and removed as expected and the buttons are reflecting the state of the selected text. make sure to try different list options, toggling those back and forth and applying styling within lists, etc.
- try using the back and forward buttons before adding any text or styling, they should be disabled. add some styling and click back until you reach the original state and the button is disabled, then do the opposite with the redo button.
- try using the clear formatting button and confirm it works as expected
- copy a url to your clipboard, select a piece of text and click the link button - the link should be applied. does this seem like an adequate way of implementing links or do we need something a bit more user friendly?
- try copying an email or some other previously formatted text (make sure it also has hyperlinks in it) and pasting it into the window
- save a note and make sure it appears correctly below and that the editor is cleared after a successful save
- edit an existing note, the editor should appear in the note space and the note should appear as it did in the ui. the toolbar buttons will be a lighter color than when you are adding a new note. does that seem okay or do we need to override that behavior?
- try editing a note that was made using quill, does it appear as expected? can you edit and save it?
- inspect the notes in the backend. do the tags, styles, etc. look okay?

---
#### Ship list
~- [ ] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~
